pkg/controllers/consolidation/controller.go

```diff
@@ -456,7 +474,10 @@ func (c *Controller) nodeConsolidationOptionReplaceOrDelete(ctx context.Context,
 			schedulableCount += len(inflight.Pods)
 		}
 		if len(node.pods) == schedulableCount {
-			savings := node.instanceType.Price()
+			savings, err := getNodePrice(node)
```

Comment:
Reading through the code, I'm struggling to see where savings is even used.

Reply:
Actually a good question. @tzneal what was the intention of this `.savings` value? It looks like this value is currently not getting used anywhere in the code

Reply:
At one point you could configure it to consolidate based on maximizing savings or minimizing cluster disruption. It's fine to remove it.